config/api.yml

```diff
@@ -750,6 +751,10 @@
       :get:
       - :name: read
         :identifier: container_image_show
+    :subresource_actions:
+      :post:
+      - :name: scan
+        :identifier: container_image_scan
```

Comment:
shouldn't we have the section:

```
:get:
- :name: read
  :identifier: container_image_show
```

in the subresource_actions section to allow one to do GET /api/providers/:id/container_images/:ci_id ?